Registry/RegistryParse.py

```diff
@@ -288,19 +346,19 @@ def hive_name(self):
     def first_hbin_offset(self):
         """
         Get the buffer offset of the first HBINBlock as an unsigned integer.
-        Note: always returns 0x1000 nothing else is known.
+        Note: always returns 0x1000, nothing else is possible.
         """
         return 0x1000
 
-    def last_hbin_offset(self):
```

Comment:
Ok, then for compability reasons (just in case other scripts refers to `last_hbin_offset()`) I would recommend adding:
```Python
     def last_hbin_offset(self):
         """
         Obsolete, returns hbin size, not the offset, use properly named hbins_size instead.
         """
          from warnings import warn
          warn("records is obsolete, use cells instead!")
          return self.hbins_size()```

A similar function for the old `validate()` behavior should be provided below